src/app/backend/dashboard.go

```diff
@@ -86,7 +89,17 @@ func main() {
 	// TODO(maciaszczykm): Move to /appConfig.json as it was discussed in #640.
 	http.Handle("/api/appConfig.json", handler.AppHandler(handler.ConfigHandler))
 	http.Handle("/metrics", prometheus.Handler())
-	log.Print(http.ListenAndServe(fmt.Sprintf("%s:%d", *argBindAddress, *argPort), nil))
+
+	// Listen for http and https
+	addr := fmt.Sprintf("%s:%d", *argBindAddress, *argPort)
+	if argBindAddress.Equal(net.IPv4(127, 0, 0, 1)) {
```

Comment:
or make two variables --insecure-bind-address and --bind-address

similar to apiserver:
https://kubernetes.io/docs/admin/kube-apiserver/